content/blog/2022/03/2022-03-25-systemd-migration.adoc

```diff
@@ -0,0 +1,156 @@
+
+Users have also been requesting `systemd(1)` support in link:https://issues.jenkins.io/browse/JENKINS-41218[JENKINS-41218] since January 2017.
+
+The migration to `systemd(1)` eliminates this long-standing pain point by relying on `systemd(1)` to daemonize the Jenkins Java process. This avoids the need for JNA-based tricks in Java, and it also unifies our implementation of service management across all Linux distributions. Furthermore, it allows for tighter integration between Jenkins core and the service management framework with regard to service startup notification.
+
+=== Changes
+
+Although `systemd(1)` has been the default service management framework in the majority of Linux distributions for seven years, some members of our community may not be familiar with its use in the context of managing Jenkins.
+The following section describes some of the changes in more detail and explains how to perform common system administration tasks.
+For up-to-date information, refer to the link:/doc/book/system-administration/systemd-services/[Managing systemd services] page in the documentation.
+See also the link:https://www.digitalocean.com/community/tutorials/how-to-use-systemctl-to-manage-systemd-services-and-units[DigitalOcean community `systemd(1)` tutorial] to better understand the benefits of `systemd(1)` and the `systemctl(1)` command.
+
+==== Package configuration
+
+Users who run Jenkins 2.335 or later and 2.332.1 LTS or later through the official Debian, Red Hat, or openSUSE packages are affected by this migration.
+
+NOTE: The link:https://hub.docker.com/r/jenkins/jenkins[official Docker image] and link:https://charts.jenkins.io/[Helm chart] remain unchanged.
+
+The System V `init(8)` packages delivered the following configuration files:
+
+Debian:: `/etc/default/jenkins`
+Red Hat:: `/etc/sysconfig/jenkins`
+openSUSE:: `/etc/sysconfig/jenkins`
+
+These configuration each used a unique format, with the Debian version being radically different from the Red Hat and openSUSE versions.
+Furthermore, manual edits to these files would frequently conflict with updated upstream versions, triggering the package manager to flag the conflict and requiring manual intervention on upgrade.
+
+New versions of Jenkins still ship these files and support Linux distributions that do not use `systemd(1)`.
+At the time, no actively-supported distributions based on Red Hat or openSUSE are known to use System V `init(8)`, leaving the `/etc/sysconfig/jenkins` file in a vestigial state and unused at runtime.
```

Comment:
Given I had to google the word and still don't understand what it means I think this should be simplified, suggested wording
```suggestion
At the time, no actively-supported distributions based on Red Hat or openSUSE are known to use System V `init(8)`, the `/etc/sysconfig/jenkins` file is left in place and unused at runtime.
```

Reply:
I agree that `vestigial` is better replaced by another term for our multi-language, multi-continent, and multi-culture audience.

Reply:
"Vestigial" just means an organ that no longer has a purpose but nevertheless has survived evolution, like the appendix. And encountering words that one does not know, looking them up, and learning is a great way to improve one's vocabulary. At least that is how I have improved mine.

In general I dislike making writing decisions based on the assumed level of readers' vocabulary, and the suggestion contains a [comma splice](https://www.grammarly.com/blog/comma-splice/), but in this particular case it seems like it would be especially easy to be confused, so I have reworded this in commit c0cf522dd.